modules/react/layout/stories/Box.stories.mdx

```diff
@@ -1,5 +1,5 @@
 import {Meta} from '@storybook/addon-docs';
-import {Box} from '@workday/canvas-kit-labs-react/common';
+import {Box} from '../';
```

Comment:
I'm unsure why some imports were changed to be absolute while others were changed to be relative. Seems arbitrary, but this file will be imported by the doc site, so there can be no relative paths to packages. Only examples can be relative.

```suggestion
import {Box} from '@workday/canvas-kit-labs-react/layout';
``

Reply:
I have to clean this up, this was before i found the reason why the package path was giving me an error because i had circular dependencies

Reply:
I updated the suggestion. I noticed the path was pointing to labs.